packages/audiofileplayer/android/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
The assets should only be present for the example app right? Why would the plugin need these?

Reply:
The plugin itself (not the example app) includes a set of available Notification.Action (i.e. icons and strings for use in displaying buttons in the foreground service notification). This way, a user of the plugin can call 

`Audio.setAndroidButtons([AndroidMediaButtonType.play]);`

to show a 'play' button in the notification. The title ('play') must be provided (here as a string resource), but is not displayed in the notification due to using a MediaStyle which instead uses an icon (included in the drawable resources)

Reply:
SG this is for background playback.

I am not opinionated on this but I want to point out a couple of things to think about:
- Bundling assets with plugins is undesirable since we cannot tree shake them (an app might use your plugin but not the background playback feature).
- For use cases like foreground notification, apps typically want customization so they might want to provide their own assets.

By *requiring* apps to provide foreground notification assets during plugin setup, you can solve both problems.

Reply:
Understood.
* I understand not wanting to bloat with assets. However, I'd rather provide 7 icons and short Strings than add the friction of telling Flutter users to learn about Android resource folders and create their own assets for the most common use cases.
* Yes, users can create their own custom notification actions, if they add icons to the Android drawable folder, and use the AndroidCustomMediaButton model class. The example app shows this example of mixing both the 'built-in' buttons (described as enum values) and a custom 'replay' button (where the user has added the 'ic_replay_black_36dp' to the 'drawable' resources)
```dart
Audio.setAndroidNotificationButtons(<dynamic>[
      AndroidMediaButtonType.pause,
      AndroidMediaButtonType.stop,
      const AndroidCustomMediaButton(
          'replay', replayButtonId, 'ic_replay_black_36dp')
    ], androidCompactIndices: <int>[
      0
    ]);
```